simtools/constants.py

```diff
@@ -0,0 +1,7 @@
+"""Project wide constants."""
```

Comment:
Lot's of good points and should discuss this. As mentioned in the pull request text, there are several solution proposed in the internet and this one is widely accepted. However, we need to make sure that this is a good solution.

- it should be for any constants, not directories only. So I guess the name is right. If you know any other constant, let's discuss if we want to put them here.
- the files are essential for running simtools and are based on CTA standards. I don't think it is configuration (plus we don't have a configuration directory). Also, since they are essential, they need to be packaged also with pip and therefore should be in the simtools directory.
- `utils/names.py` does something different: it builds and validates names (mostly based on instrument classes, sites, etc).

Reply:
- Ok for the name;
- There is a `simtools/configuration` directory but I think that is meant for application configuration only, right?
- `utils/names.py` has also the string definitions for 'mst', 'sst', 'lst', for example. I think such definitions would fit better in the new `constants.py` and then we would leave `utils/names.py` only for validating names (just an idea to follow up in a forthcoming PR?)

Reply:
For names.py, see issue #224 (and might be covered by schemas in future). Configuration is handling indeed the application configuration. Both are doing very different things to constant.py.